pkg/actors/actors.go

```diff
@@ -497,7 +497,7 @@ func (a *actorsRuntime) GetState(ctx context.Context, req *GetStateRequest) (*St
 
 func (a *actorsRuntime) TransactionalStateOperation(ctx context.Context, req *TransactionalRequest) error {
 	if a.store == nil || a.transactionalStore == nil {
-		return errors.New("actors: state store does not exist or incorrectly configured")
+		return errors.New("actors: state store does not exist or incorrectly configured. Have you set the - name: actorStateStore value: \"true\" in your state store component file?")
```

Comment:
Shouldn't this return an error if Entities > 0?

Reply:
Actors are using state store but actor state store does not exist, regardless of the Entities count, the error should be always returned. WDYT?